pennylane/capture/__init__.py

```diff
@@ -114,4 +153,18 @@ def _(*args, **kwargs):
 """
 from .switches import disable, enable, enabled
 from .capture_meta import CaptureMeta
-from .primitives import create_operator_primitive
+from .primitives import (
+    create_operator_primitive,
+    create_measurement_obs_primitive,
+    create_measurement_wires_primitive,
+    create_measurement_mcm_primitive,
+)
+from .measure import measure
+
+
+def __getattr__(key):
+    if key == "AbstractOperator":
+        from .primitives import _get_abstract_operator  # pylint: disable=import-outside-toplevel
+
+        return _get_abstract_operator()
+    raise AttributeError(f"module 'pennylane.capture' has no attribute '{key}'")
```

Comment:
Sneaky!

Reply:
Potentially overcomplicating all the lazy loading to hide the jax dependency, but this seems to work for now.